test/test_supervisor.rb

```diff
@@ -1054,6 +1054,129 @@ def test_stop_parallel_old_supervisor_after_delay
     end
   end
 
+  sub_test_case "include additional configuration" do
+    setup do
+      @config_include_dir = File.join(@tmp_dir, "conf.d")
+      FileUtils.mkdir_p(@config_include_dir)
+    end
+
+    test "no additional configuration" do
+      c = Fluent::Config::Element.new('system', '', { 'config_include_dir' => '' }, [])
+      stub(Fluent::Config).build { config_element('ROOT', '', {}, [c]) }
+      supervisor = Fluent::Supervisor.new({})
+      stub(supervisor).build_spawn_command { "dummy command line" }
```

Comment:
```suggestion
```

Since this test does not call `Supervisor#run_supervisor`, we don't need to stub `Supervisor#build_spawn_command`.